foyer/forcefield.py

```diff
@@ -125,6 +125,7 @@ def _topology_from_parmed(structure, non_element_types):
                 element = elem.Element.getBySymbol(pmd_atom.name)
 
         omm_atom = topology.addAtom(name, element, residues[pmd_atom.residue])
+        omm_atom.id = pmd_atom.id
```

Comment:
Per our conversation earlier, I think `omm_atom.id` refers to the atom type.  a ParmEd `Atom` doesn't have an `id` attribute and we should be calling `type` instead.